.github/workflows/web.yml

```
@@ -35,7 +35,7 @@ jobs:
               base=/atip/$branch
             fi
 
-            if npm run wasm-release && npm ci && npm run build --if-present -- --base=$base; then
+            if npm run wasm-release && npm run compile-styles && npm ci && npm run build --if-present -- --base=$base; then
```

As noted, this'll break on every single branch except this new one. How about we run this separately above and allow it to fail? After merging this PR and rebasing all others, we can make it part of the mandatory step.

But also, the script uses `sass`, which'll only be installed after doing `npm ci`. So we should invert the order in the `&&`s, and in the temporary version

```suggestion
            npm run ci; npm run compile-styles
            if npm run wasm-release && npm ci && npm run build --if-present -- --base=$base; then
```